### PR TITLE
Add pipeline to run test in docker image.

### PR DESCRIPTION
**Pull request by manuelsaldivar525**
### Description
Add pipeline to run test into docker image
### Related GitHub issue or JIRA story
https://jira.autodesk.com/browse/DYN-2074
https://jira.autodesk.com/browse/DYN-2075
### Checklist
- [ ] PR title reflects the changes it introduces. Consider using this template: QNTM-XXXX: Implement X Feature
- [ ] Code compiles correctly
- [ - ] No linting errors
- [ - ] All tests passing
- [ ] Updated README.md / documentation (if necessary)

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+@Library('PSL@master')
+@Library('CILibrary@CBP/DynamoPerfTest') _
```
Comment (QilongTang) [inline Jenkinsfile]: @manuelsaldivar525 Is there a way to add some comments in Jenkins file to describe some more why these two are the first two entries?

File: pipeline.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+version: 0.1.1
```
Comment (QilongTang) [inline pipeline.yml]: Should this be incremented everytime who touches this?

File: Jenkinsfile (hunk 1 above)
Comment (manuelsaldivar525) [reply Jenkinsfile]: I added like in documentation sample, the documentation of https://git.autodesk.com/Quantum/CILibrary I mean. 
In fact the documentation includes 
#!/usr/bin/env groovy

@Library('PSL@master')
@Library('CILibrary@CBP/stable') _

StartPipeline()

File: pipeline.yml (hunk 2 above)
Comment (manuelsaldivar525) [reply pipeline.yml]: No, I added like in documentation sample, the documentation of https://git.autodesk.com/Quantum/CILibrary I mean.

File: Jenkinsfile (hunk 1 above)
Comment (mjkkirschner) [reply Jenkinsfile]: I think Aaron meant something like:

`// These import the groovy libraries required to build this repo.`

**Comment by mjkkirschner** [comment]
regarding the npmrc file - I believe after this is committed that devs outside adsk will not be able to build the repo without removing this line - can we also add that to the documentation if found to be true @jesusHCG?

**Comment by jesusHCG** [comment]
Delete the node_modules folder and execute the commands disconnected from the VPN and they were executed without problems. But I will add a note in the documentation

**Comment by mjkkirschner** [comment]
@jesusHCG well thats surprising, maybe it's using locally cached node modules - https://docs.npmjs.com/cli/cache.html

File: README.md [hunk 3]
```diff
@@ -6,6 +6,8 @@ Installing all dependencies
 
     $ npm install
 
+Note: In some cases it may be required to comment or remove the contents of the .npmrc file to install the dependencies.
```
Comment (mjkkirschner) [inline README.md]: can you change this to - if you are having trouble running `npm install` and are outside the adsk network, please try removing the `.npmrc` file.

**Comment by mjkkirschner** [comment]
LGTM

**Pull request merged**